ibis/tests/benchmarks/test_benchmarks.py

```diff
@@ -850,19 +850,20 @@ def test_column_access(benchmark, many_cols, getter):
     benchmark(getter, many_cols)
 
 
-@pytest.fixture(scope="module")
-def many_tables():
+@pytest.fixture(scope="module", params=[1000, 10000])
```

Comment:
A table with 10k unions is definitely hitting the upper limit of interactive latency acceptability (about 9s on a gcp instance with 64 cores and 96G of RAM).